tests/conftest.py

```diff
@@ -14,7 +34,8 @@ def testing_images():
     async def _pull():
         docker = Docker()
         required_images = [
-            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine'
+            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine',
+            'busybox:latest',
```

Comment:
Yes, we could simply use an image that runs "sleep 60", that's a container service that will be running and that all we actually need for the test.